tests/test_scrape.py

```diff
@@ -33,12 +33,13 @@
     SmokeTestLanguage("kor", "Korean", {"phonetic": True}),
     SmokeTestLanguage("khm", "Khmer", {}),
     SmokeTestLanguage("tha", "Thai", {}),
+    SmokeTestLanguage("lat", "Latin", {}),
 ]
 
 
 @pytest.mark.skipif(not can_connect_to_wiktionary(), reason="need Internet")
 @pytest.mark.parametrize("smoke_test_language", _SMOKE_TEST_LANGUAGES)
-@pytest.mark.timeout(3)
+@pytest.mark.timeout(4)
```

Comment:
I could get the Latin smoke test to pass the smoke test under 3 seconds for timeout. I'm writing up a ticket with some thoughts about performance in general.